app/jobs/validation_retry_job.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class ValidationRetryJob < CronJob
+  self.cron_expression = "15 3 * * *"
+
+  queue_as :validation_retry
+
+  def perform
+    ECFParticipantValidationData.where(api_failure: true).each do |validation_data|
+      validation_result = ParticipantValidationService.validate(trn: validation_data.trn,
+                                                                full_name: validation_data.full_name,
+                                                                date_of_birth: validation_data.date_of_birth,
+                                                                nino: validation_data.nino)
+      ActiveRecord::Base.transaction do
+        validation_data.update!(api_failure: false)
+        if validation_result.present?
+          eligibility_data = store_eligibility_data!(validation_data, validation_result)
+          eligibility_data.manual_check_status! unless store_trn!(validation_result[:trn], validation_data.participant_profile)
+
+          validation_data.destroy! if eligibility_data.eligible_status?
+        end
+      end
+    rescue StandardError => e
+      Rails.logger.error("Problem with DQT API on retry: " + e.message)
+      Sentry.capture_message("Problem with DQT API on retry: " + e.message)
```

Comment:
Do we need both?

Reply:
I don't think it does any harm

Reply:
I am just worried that in other places when we capture sentry we don't explicitly log an error in rails. It would be nice to be consistent.

Is there some way to tell Sentry to rails log the errors it catches?